### PR TITLE
PCL (portable class library) support

### PR DESCRIPTION
**Pull request by stefaneberl**

### CODE REVIEW

File: src/openkit-shared/Util/IEnumerableExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace Dynatrace.OpenKit.Util
+{
+
+#if NETPCL4_5
+
+    /// <summary>
+    /// Utility class providing some extension methods which are not available in certain
```
Comment (y0chanan) [inline src/openkit-shared/Util/IEnumerableExtensions.cs]: missing text at the end

File: src/openkit-dotnetpcl-4.5/MyClass.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
```
Comment (y0chanan) [inline src/openkit-dotnetpcl-4.5/MyClass.cs]: is this class needed or will it be extended later

**Review by y0chanan — CHANGES_REQUESTED** [review]

File: src/openkit-dotnetpcl-4.5/MyClass.cs (hunk 2 above)
Comment (stefaneberl) [reply src/openkit-dotnetpcl-4.5/MyClass.cs]: Nope, leftover when generating a new project.

File: src/openkit-shared/Util/IEnumerableExtensions.cs (hunk 1 above)
Comment (stefaneberl) [reply src/openkit-shared/Util/IEnumerableExtensions.cs]: Changed

File: src/openkit-dotnetpcl-4.5/MyClass.cs (hunk 2 above)
Comment (stefaneberl) [reply src/openkit-dotnetpcl-4.5/MyClass.cs]: File has been removed.

**Review by thomasgrassauer — APPROVED** [review]

**Review by y0chanan — APPROVED** [review]

**Pull request merged**